worlds/noita/Items.py

```diff
@@ -0,0 +1,143 @@
+import itertools
+from typing import Dict, NamedTuple, Optional, List, Set
+from BaseClasses import Item, ItemClassification, MultiWorld
+from .Options import BossesAsChecks, VictoryCondition
```

Comment:
Might be clearer to use fully qualified `Options.whatever` below, same elsewhere.

Reply:
Decided against it